lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -897,7 +897,8 @@ LogicalResult FIRRTLModuleLowering::lowerPorts(
     hw::PortInfo hwPort;
     hwPort.name = firrtlPort.name;
     hwPort.type = lowerType(firrtlPort.type);
-    hwPort.sym = firrtlPort.sym;
+    hwPort.sym = firrtlPort.sym ? firrtlPort.sym.getSymName()
+                                : StringAttr::get(moduleOp->getContext(), "");
```

Comment:
Could you emit warnings if firrtlPort has field sensitive symbols? (since we are currently dropping field sensitive symbols at LowerToHW)

Reply:
Yes, that and the I need to create an issue. Lets schedule a meeting this week and discuss the path forward here.

Reply:
Thank you for emitting errors!